301-web-app-regional-vnet-private-endpoint-sql-storage/nestedtemplates/private_link_ipconfigs_helper.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "privateDnsZoneName": {
+      "type": "string",
+            "metadata": {
+        "description": "private DNS zone name"
+      }
+    },
+    "privateLinkNicIpConfigs": {
+      "type": "array",
+      "metadata": {
+        "description": "private endpoint nic ip configuration array"
+      }
+    },
+    "privateDnsRecordTemplateUri": {
+      "type": "string",
+      "metadata": {
+        "description": "private dns zone record template resource URI"
+      }
+    }
+  },
+  "variables": {
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2019-10-01",
+      "name": "[concat('nestedTemplate-private-link-ipconfigs-helper', copyIndex())]",
+      "properties": {

```

Comment:
move properties to the bottom of the resource definition

Reply:
Ok, I have made the requested change.